lib/libfrr.h

```diff
@@ -129,6 +129,7 @@ struct frr_daemon_info {
 	bool daemon_mode;
 	bool terminal;
 	bool graceful_restart;
+	bool tls_mode;
```

Comment:
too generic - this is log-specific, right, not eliminating any TLS of any kind anywhere?